packages/tools/kolibri/package.json

```
@@ -0,0 +1,54 @@
+{
+	"name": "@public-ui/kolibri",
+	"version": "1.7.0-rc.0",
+	"license": "EUPL-1.2",
+	"homepage": "https://public-ui.github.io",
+	"repository": "https://github.com/public-ui/kolibri",
+	"bugs": {
+		"url": "https://github.com/public-ui/kolibri/issues",
+		"email": "kolibri@itzbund.de"
+	},
+	"author": {
+		"name": "Informationstechnikzentrum Bund",
+		"email": "kolibri@itzbund.de"
+	},
+	"bin": {
+		"kolibri": "dist/index.js"
+	},
+	"description": "Console tool for executing some helpful commands for KoliBri projects.",
```

```suggestion
	"description": "CLI for executing some helpful commands for KoliBri projects.",
```
Vorschlag, klingt vielleicht griffiger.